packages/compute/src/vm.js

```diff
@@ -45,6 +45,24 @@ var DetachDiskError = createErrorClass('DetachDiskError', function(message) {
   this.message = message;
 });
 
+/**
+ * Interval for polling during waitFor.
+ */
+var WAIT_FOR_POLLING_INTERVAL = 2000;
```

Comment:
Should this be a cadence, or a backoff algorithm?
There is no API or rate limit involved, so a candence might be entirely fine.